libs/feature/search/src/lib/state/search.facade.ts

```diff
@@ -60,6 +60,11 @@ export class SearchFacade {
   constructor(private store: Store<SearchState>) {}
 
   init(searchId: string = DEFAULT_SEARCH_KEY): void {
+    if (this.searchId)
```

Comment:
does it really matters ?
I mean, how do you interact with a search state if you are outside of the `search-directive` DOM tree ?

Reply:
it would have thrown an error before this PR since we were initializing the same facade multiple times, which could happen in the future (nothing else prevents it)

Reply:
still you don't answer my question.
it's not that a big deal (even though it's not clean) to have several facades with the same id, in the end the id is just used to target to correct state. If your facade is encapsulated in a DOM tree by the directive, then you can't interact with the state from outside this DOM section (via the facade at least because you can still dispatch some action with the search id but it's not meant to be like that).

let's discuss that later

Reply:
the problem that this change is trying to avoid is that a SearchFacade instance is reused by mistake in several places, even though the developer assumed he/she was using different ones; this is exactly what happened to me in the datahub